cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -65,6 +66,8 @@ std::optional<nvcompStatus_t> batched_decompress_get_temp_size_ex(compression_ty
 #else
       return std::nullopt;
 #endif
+    case compression_type::LZ4:
```

Comment:
Does this need to be gated by some version support macro?

Reply:
no need the check the version, LZ4 is in all nvcomp versions we support (same situation as with snappy)